src/main/java/core/basesyntax/service/FileValidator.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax.service;
+
+import java.util.List;
+
+public class FileValidator implements Validator {
+    @Override
+    public void validate(List<String> lines) {
+        for (int i = 1; i < lines.size(); i++) {
```

Comment:
it seems identical to your loop in main()

maybe let's make this service validate only 1 line?